src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php

```diff
@@ -52,6 +53,8 @@
  */
 abstract class AbstractCategoryType extends TranslatorAwareType
 {
+    private const MAX_TITLE_LENGTH = 128;
```

Comment:
Any reason to use private? Cannot be used in children classes :(

Reply:
Seems to be a duplicate from SeoSettings::MAX_TITLE_LENGTH

Reply:
Yeah it should be public, but it's not duplicate. The one from SeoSettings is meant for meta title and has different length then actual title.